docs/customizations.md

```diff
@@ -0,0 +1,233 @@
+# Grow Up Story for Generated SDKs
+
+This quickstart will introduce how to grow up your generated code with customizations.
+
+**Note: This quickstart does not focus on renaming or removing generated objects**. If you'd like to rename or remove generated objects, the changes should really be made in the
+OpenAPI definition of your service. If you absolutely can not modify the OpenAPI definition, you can use directives, specifically [these pre-defined directives](https://github.com/Azure/autorest/blob/main/docs/generate/built-in-directives.md) in your README configuration
+
+## Key Concept: _patch.py
+
+The `_patch.py` files at each level of the modules will be the entry point to customize the generated code.
```

Comment:
Technically, I think you mean "packages" (https://docs.python.org/3/tutorial/modules.html#packages), not "modules". But being technically correct here may actually confuse people. Opinions?

Reply:
let's go with subfolder